packages/stack/proxy/src/proxy.js

```diff
@@ -152,7 +153,9 @@ export class Proxy {
       if (modifiedResp && modifiedResp.response && modifiedResp.response.error) {
         // error returned from the node and it wasn't stripped by our response actions
         const error = modifiedResp.response.error.message || modifiedResp.response.error;
-        this.logger.error(__(`Error returned from the node: ${error}`));
+        if (!this.context || (this.context.length && !this.context.includes(constants.contexts.test))) {
```

Comment:
I'm wondering if maybe we should log it in `debug` or `trace` mode, instead of not logging, so that if users have issues they can still get access to that log if needed?

Reply:
I agree, and @emizzle suggested the same.